dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -1816,6 +1820,16 @@ Spdp::SpdpTransport::close()
   reactor_task_.stop();
 }
 
+void
+Spdp::SpdpTransport::bump_up_local_write_i()
```

Comment:
I would rename this function to something more descriptive like `shorten_local_sender_delay`.